vagrant/env.sh

```diff
@@ -12,5 +12,9 @@ export TREEHERDER_DJANGO_SECRET_KEY='secret-key-of-at-least-50-characters-to-pas
 export NEW_RELIC_CONFIG_FILE='newrelic.ini'
 export NEW_RELIC_DEVELOPER_MODE='True'
 
+# Auth0 Config
+export AUTH0_DOMAIN='auth.mozilla.auth0.com'
+export AUTH0_AUDIENCE='login.taskcluster.net'
```

Comment:
`settings.py` has defaults for these, so these can probably be removed (unless we need to vary them more per environment)